src/test/java/com/adyen/BaseTest.java

```diff
@@ -229,6 +233,8 @@ protected PaymentsRequest createAfterPayPaymentRequest() {
                 .taxAmount(52L)
                 .amountIncludingTax(300L)
                 .taxCategory(LineItem.TaxCategoryEnum.HIGH)
+                .imageUrl("dummy_protocol/image_url/")
```

Comment:
Why are we using the string here when we have the constant ?

Reply:
fixed. pushed.